Moodis/Moodis/Ui/MenuMethods.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Moodis.Network.Face;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace Moodis.Ui
+{
+    class MenuMethods
+    {
+        public static ImageInfo currentImage = new ImageInfo();
+        private static Bitmap userImage;
+        private static string jsonAsString;
+
+        public static Bitmap ShowImage(String fileToDisplay)
```

Comment:
This should not be static, make it a simple method callable from MenuViewModel object instance